contributing.md

```diff
@@ -42,14 +42,14 @@ more before applying. When contacting us, please include a link to your existing
 ## Developing `electron-i18n`
 
 The rest of this document describes how this repo is structured, how
-it syncs with Crowdin, and how to make technical changes.
+it syncs with Crowdin, how to make technical changes, and how to configure the proofreader role properly.
 
 **If you're just here to translate, you probably don't need to read this.**
 
 ### Environment Setup
 
 To fetch the latest docs, you need a GitHub token. Visit
-github.com/settings/tokens/new](https://github.com/settings/tokens/new)
+[https://github.com/settings/tokens/new](https://github.com/settings/tokens/new)
```

Comment:
The protocol is not needed in the link text. it's cleaner without.

Reply:
it makes sense.